docs/aca/00-workshop-intro/4-prerequisites.md

```diff
@@ -13,6 +13,7 @@ Make sure you have your development environment set up and configured.
 
 1. An Azure account with an active subscription - [Create an account for free](https://azure.microsoft.com/free/?ref=microsoft.com&utm_source=microsoft.com&utm_medium=docs&utm_campaign=visualstudio){target=_blank}
 1. dotnet 6.0 or a higher version - [Install](https://dotnet.microsoft.com/en-us/download){target=_blank}
+1. PowerShell 7.0 or higher version - [Install](https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-windows?view=powershell-7.4#installing-the-msi-package){target=_blank}
```

Comment:
It will be great to mention only when using windows, kindof obvious, but still it will be good to make it explicit.